os-autoinst-obs-auto-submit

```diff
@@ -7,7 +7,7 @@ src_project=${src_project:-devel:openQA}
 dst_project=${dst_project:-${src_project}:tested}
 staging_project=${staging_project:-${src_project}:testing}
 submit_target="${submit_target:-"openSUSE:Factory"}"
-submit_target_extra="${submit_target_extra:-}"
+submit_target_extra="${submit_target_extra:-"openSUSE:Backports:SLE-15-SP6:Update"}"
```

Comment:
Did you try this manually?

I'm asking since we've had broken Jenkins jobs for multiple days related to the original change.

Reply:
Or if it's not trivial to execute manually, I guess we just need to merge within a working day and consider reverting quickly.

Reply:
I didnt try manual. I can say that I didnt see the underlying problem from jenkins and it was not expected to be seen because the part which causes it, it wont run against the Backports target anymore. ofc you can go also with the `we just need to merge within a working day and consider reverting quickly.` in any case. But I would say I am quite confident.

Reply:
We know that it was failing repeatedly and there's no unit test coverage. Regardless of how that happened, that's the reason I asked 🙃

Let's merge it early on Friday then.